validation-test/BuildSystem/skip_clean_corelibs.test

```diff
@@ -1,9 +1,5 @@
 # REQUIRES: standalone_build
 # UNSUPPORTED: OS=macosx
-# UNSUPPORTED: OS=ios
-# UNSUPPORTED: OS=tvos
-# UNSUPPORTED: OS=watchos
-# UNSUPPORTED: OS=xros
```

Comment:
@artemcm, these build system tests were disabled on remote CI like iOS or Android two years ago, but you disabled xros after that: are these tests being run natively on xros, so I need to keep this disabled?

Reply:
Never mind, looks like these are still run for the iPhone simulator, so reverting this change of platforms for this one test.